src/main_menu.cpp

```diff
@@ -99,26 +101,27 @@ void main_menu::print_menu( const catacurses::window &w_open, int iSel, const in
     const nc_color cColor2 = c_light_blue;
     const nc_color cColor3 = c_light_blue;
 
+    std::string marker = "";
```

Comment:
You don't need to initialize a string with an empty C-String. It's automatically created as empty string.